lib/mvi/service.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 require 'savon'
+require_relative 'response'
+require_relative 'mock_service'
```

Comment:
It's not used by the real service but other locations that require the real service might be using the mock one, might be a better pattern to handle this.